### PR TITLE
Data Release 20240410

### PR DESCRIPTION
**Pull request by hareesh-ms**

### CODE REVIEW

File: internal/server/v2/propertyvalues/golden/triple_in/result1.json [hunk 1]
```diff
@@ -2,19 +2,6 @@
   "data": {
     "BiologicalSpecimen": {
       "arcs": {
-        "subClassOf": {
-          "nodes": [
-            {
-              "name": "Virus",
```
Comment (hareesh-ms) [inline internal/server/v2/propertyvalues/golden/triple_in/result1.json:8]: Part of Biomedical cleanup cl/622988325

File: internal/server/v0/placestatvar/golden/get_place_stat_vars/santa_clara.json [hunk 2]
```diff
@@ -4990,9 +4990,13 @@
         "Count_MortalityEvent_YearsUpto1_ICD10/P00-P96_AsAFractionOf_Count_Person_YearsUpto1",
         "Count_MortalityEvent_YearsUpto1_ICD10/Q00-Q99_AsAFractionOf_Count_Person_YearsUpto1",
         "Count_Person",
+        "Count_Person_0To4Years_AAC_FEMALE",
```
Comment (ajaits) [inline internal/server/v0/placestatvar/golden/get_place_stat_vars/santa_clara.json:4993]: Not a blocker, but I don;t see these Statvars in the schema repo. 

Can we check which import this is coming from?

**Review by ajaits — APPROVED** [review]

File: internal/server/v0/placestatvar/golden/get_place_stat_vars/santa_clara.json (hunk 2 above)
Comment (hareesh-ms) [reply internal/server/v0/placestatvar/golden/get_place_stat_vars/santa_clara.json:4993]: This is coming from the US Census PEP import. Will check the recent update CL changes/

**Review by n-h-diaz — APPROVED** [review]
Thanks!

File: internal/server/v1/variables/golden/bulk_variables/california_and_santa_clara_union.json [hunk 3]
```diff
@@ -9381,6 +9397,18 @@
...
+        "Count_Person_0To4Years_NHAAC_FEMALE",
+        "Count_Person_0To4Years_NHAAC_MALE",
+        "Count_Person_0To4Years_NHBAC_FEMALE",
+        "Count_Person_0To4Years_NHBAC_MALE",
+        "Count_Person_0To4Years_NHIAC_FEMALE",
+        "Count_Person_0To4Years_NHIAC_MALE",
+        "Count_Person_0To4Years_NHNAC_FEMALE",
+        "Count_Person_0To4Years_NHNAC_MALE",
+        "Count_Person_0To4Years_NHWAC_FEMALE",
+        "Count_Person_0To4Years_NHWAC_MALE",
```
Comment (pradh) [inline internal/server/v1/variables/golden/bulk_variables/california_and_santa_clara_union.json:9411]: What are these variables?  I'm not finding them on autopush

**Pull request merged**